src/subscriptions_handler.py

```diff
@@ -0,0 +1,86 @@
+import json
+import logging
+import os
+
+import boto3
+
+STAGES = ['TEST', 'QA', 'PROD-EXTERNAL']
+
+STAGE = os.getenv('STAGE', 'TEST')
+
+log_level = os.getenv('LOG_LEVEL', logging.ERROR)
+logger = logging.getLogger(__name__)
+logger.setLevel(log_level)
+
+secrets_client = boto3.client('secretsmanager', os.getenv('AWS_DEPLOYMENT_REGION', 'us-west-2'))
+
+
+def manage_subscriptions(event, context):
+    """
+    Obtain a list of subscribers for terminal errors and a list of subscribers for warnings from the secrets
+    manager.  Process these lists to synchronize them with the subscriptions AWS knows about.
+    """
+    original = secrets_client.get_secret_value(
```

Comment:
I might have opted for the parameter store for this, but secrets manager is fine too.